src/lib/interfaces/profile-deployment.ts

```diff
@@ -28,7 +27,6 @@ export interface ProfileDeploymentOptions {
   };
 }
 export interface DeployedContracts {
-  ERC725Account?: ERC725Account;
```

Comment:
@CallumGrindle I fixed the build error, this should not be used anymore, can you double check? (Just resolve this if OK for you, thanks)